runtime/oti/j9consts.h

```diff
@@ -950,6 +950,10 @@ extern "C" {
 #define PREVIEW_MINOR_VERSION 65535
 #define J9_IS_CLASSFILE_OR_ROMCLASS_VALUETYPE_VERSION(classfileOrRomClass) (((classfileOrRomClass)->majorVersion >= VALUE_TYPES_MAJOR_VERSION) && (PREVIEW_MINOR_VERSION == (classfileOrRomClass)->minorVersion))
 
+/* java.lang.Field value type constants */
```

Comment:
I suggest to change this comment to "Constants for java.lang.Field flags". `TRUST_FINAL` is a flag that can be applied to non-value types.